app/controllers/messages_controller.rb

```diff
@@ -3,13 +3,16 @@ class MessagesController < ApplicationController
   before_action :require_active_subscription!
 
   def create
+    @conversation = conversation
```

Comment:
You could also memoize this in your method call:

```rb
def conversation
  @conversation ||= conversation
end
```

And then you dont need to do this top level assignment!

Reply:
Ah, good call! I forgot that the next line calls into that method anyway.